contents/tutorials/migrate-eu-cloud.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How to migrate to EU Cloud
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-10-10
+featuredImage: ../images/tutorials/banners/migrate-eu-cloud.png
+topics: ['configuration']
+---
+
+For many companies, teams, and products, keeping customer data within the EU is critical for compliance. To help them with this, we’ve created an EU Cloud hosting option. It includes all the features of PostHog, with data being hosted in the EU (specifically Germany).
+
+Why this matters is that the EU has stricter data privacy laws than other areas. In the EU, there are often stricter requirements for what companies can and can’t do with customer data. For example, there is less regulation in the US on data privacy. To understand how this applies to your company and data, you should check with a legal professional.
+
+For this tutorial, we will go over how to move data you currently have in either US Cloud or self-hosting to EU Cloud.
+
+## Signing up for EU Cloud
+
+The first step in migrating to EU Cloud is having an EU Cloud PostHog instance. Even if you have an existing PostHog Cloud instance, you need to create a new one. It is not possible (right now) to change your account between US Cloud (all cloud instances before this launch) or self-hosting and EU Cloud.
+
+To start with EU Cloud, you can go to the [cloud signup page](https://app.posthog.com/signup), enter your details, and be sure to select European Union as your data region.
+
+![EU Cloud signup](../images/tutorials/migrate-eu-cloud/signup.png)
+
+## Using Migrator 3000
+
+Once you’ve created your EU Cloud account, you can immediately start migrating data there. To do this, in your original PostHog instance (either Cloud or self-hosted), go to Apps and search for the [Migrator 3000 app](https://posthog.com/docs/apps/migrator-3000). This app moves data from one PostHog instance to another. Once you've found and added it, click the blue “Configure” button to start.
+
+![Migrator 3000 app](../images/tutorials/migrate-eu-cloud/migrator-app.png)
+
+In the configuration menu, enter the project key and host (`https://eu.posthog.com`) from your EU Cloud instance settings. If you don’t want to import all the events, you can also set a date limit.
+
+![Migrator 3000 config](../images/tutorials/migrate-eu-cloud/migrator-config.png)
+
+After clicking save, the Migrator 3000 app will be enabled and start running. You can view its progress in Logs (three points icon).
+
+## Finished complete migration
+
+Once the Migrator 3000 is finished, you’ll have your data and events in your new EU Cloud instance. There are tasks you need to complete to fully migrated.
+
+1. Ensure the events you want are there. Check that EU Cloud stats match with your self-hosting or US Cloud instance.
```

Comment:
I'm not sure what EU Cloud stats means?

Reply:
I mean like EU Cloud data, I'll change it.